modules/ROOT/pages/apply-configure-xml-threat-task.adoc

```diff
@@ -9,31 +9,34 @@ endif::[]
 >s|Policy Name|XML Threat Protection
 >s|Summary|Protects against malicious XML in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
 
-XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
+Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
 
-In the event Mule Runtime fails to detect an attack, you need to monitor and design your services architecture with layers of protection in addition to these policies.
+If you find that attacks on your Anypoint Platform setup are difficult to detect, design your services architecture with layers of protection in addition to JSON Threat Protection.
 
-== Configuration
+== Configuring Policy Parameters
+
+When you apply the XML Threat Protection policy to your API from the UI, the following parameters are displayed:
 
 [width="100%", cols="2,6,1,1"]
 |===
 s|Field s|Description s|Default s|Required
-|Maximum Node Depth|Specifies the maximum node depth allowed in the XML ^|0 ^|false
-|Maximum Attribute Count Per Element|Specifies the maximum number of attributes allowed for any element. Note that attributes used for defining namespaces are not counted ^|0 ^|false
-|Maximum Child Count|Specifies a limit on the maximum number of children allowed for any element in the XML document ^|0 ^|false
-|Maximum Text Length|Specifies a limit on the maximum length, in characters, of any text nodes present in the XML document ^|0 ^|false
-|Maximum Attribute Length|Specifies a limit on the maximum length, in characters, of any attributes for any element in the XML document ^|0 ^|false
-|Maximum Comment Length|Specifies a limit on the maximum number of comment characters in the XML document ^|0 ^|false
+|Maximum Node Depth|Specifies the maximum node depth of an XML document ^|0 ^|false
+|Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted ^|0 ^|false
```

Comment:
ditto
```suggestion
|Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted. ^|0 ^|false
```

Reply:
Please supply a period at the end of each of the following description texts, as in the previous two edits.

Reply:
Done! Thanks!